### PR TITLE
[UK] Update `BBCode`

### PR DESCRIPTION
**Pull request by Aniterin**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BBCode/uk.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
----
-outdated_since: 3cfbc4e75e17092cf181c6f1235bfb105666003d
-outdated_translation: true
----
+
```
Comment (Pisapou) [inline wiki/BBCode/uk.md:1]:
```suggestion
```

File: wiki/BBCode/uk.md [hunk 2]
```diff
@@ -196,7 +193,7 @@ text
 
 *Зауважте: ID користувача - це цифри після `/users/` в посиланні на профіль.*
 
-Якщо було вказано ID користувача, то текст між тегами не буде відображений, замість нього буде відображено ім'я користувача. Якщо користувач змінює своє ім'я то посилання перестає працювати
+Якщо було вказано тільки ID користувача, то треба вписати текст між тегами, він не буде відображений, замість нього буде відображено ім'я користувача. Якщо ви вкажете ім'я користувача, то після зміни ім'я посилання перестає працювати
```
Comment (Pisapou) [inline wiki/BBCode/uk.md:196]:
```suggestion
Якщо було вказано тільки ID користувача, то треба вписати текст між тегами, він не буде відображений, замість нього буде відображено ім'я користувача. Якщо ви вкажете ім'я користувача, то після зміни ім'я посилання перестає працювати.
```

File: wiki/BBCode/uk.md [hunk 3]
```diff
@@ -208,23 +205,23 @@ text
 [/list]
 ```
 
-Тег `[list]` використовується для форматування різних типів списків по всьому сайту, використовуючи (`[*]`) ви створюєте новий елемент в списку. По замовчуванню це створить простий дотований текст.
-
-Інші типи списків можуть бути створені вказуючи `TYPE`, `TYPE` підтримує:
+Тег `[list]` використовується для вставки одного із двох типів списків по всьому сайту, використовуючи (`[*]`) ви створюєте новий елемент в списку. По замовчуванню це створить простий дотований текст.
```
Comment (Pisapou) [inline wiki/BBCode/uk.md:208]:
```suggestion
Тег `[list]` використовується для вставки одного із двох типів списків по всьому сайту, використовуючи (`[*]`) ви створюєте новий елемент в списку (показано вище). По замовчуванню це створить простий дотований текст.
```

File: wiki/BBCode/uk.md [hunk 4]
```diff
@@ -208,23 +205,23 @@ text
...
 
-`1`(пронумерований),
+Якщо `TYPE` вказаний(не важливо що саме ви напишете), буде створено нумарований список.
```
Comment (Pisapou) [inline wiki/BBCode/uk.md:210]:
```suggestion
Якщо `TYPE` вказаний (не важливо що саме ви напишете), буде створено нумарований список.
```

**Review by MegaMixYT — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]
i approve

**Pull request merged**